util/timer_test.cc

```diff
@@ -0,0 +1,223 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#include "util/timer.h"
+
+#include "db/db_test_util.h"
+
+class TimerTest : public testing::Test {
+ public:
+  TimerTest()
+      : mock_env_(new ROCKSDB_NAMESPACE::MockTimeEnv(
+          ROCKSDB_NAMESPACE::Env::Default())) {}
+
+ protected:
+  std::unique_ptr<ROCKSDB_NAMESPACE::MockTimeEnv> mock_env_;
```

Comment:
Maybe i missed anything but I didn't see feature of MockTimeEnv is used here.
If the purpose is to simulate TimedWait without actual waiting, tests like StatsHistoryTest.RunStatsDumpPeriodSec does something complicated to achieve that.